src/index.html

```diff
@@ -5,10 +5,28 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+          href="https://fonts.gstatic.com" crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="logo" href="#"><img src="./images/logo.png" alt="logo"></a>
+      <nav class="nav">
+        <ul>
+          <li><a class="is-active" href="#">Apple</a></li>
```

Comment:
1. add classes
2. apply code styling recommendation 

```suggestion
        <ul class="nav__list">
          <li class="nav__item">
            <a class="is-active nav__link" href="#">
              Apple
            </a>
          </li>
```